_shared/project/pyproject.toml

```diff
@@ -89,9 +89,7 @@ jobs = 0 # Use one process for CPU.
 load-plugins = [
     "pylint.extensions.bad_builtin",
     "pylint.extensions.check_elif",
-    "pylint.extensions.comparetozero",
     "pylint.extensions.docparams",
-    "pylint.extensions.emptystring",
```

Comment:
In pylint 3.0 `emptystring` was replaced by `use-implicit-booleaness-not-comparison-to-string` below